main.go

```diff
@@ -102,21 +102,29 @@ func main() {
 
 		switch {
 		/* GIVEN A TOML FILE, */
-		/* show all terraform version including betas and RCs*/
+		/* show all terraform version including betas and RCs */
 		case *listAllFlag:
```

Comment:
I'm trying to work out why this case switch logic is duplicated for the TOML file.
Can the TOML file detection not be done before the switch case and just update the variables for version/binPath and then run the single case switch?

Reply:
Though, I'm happy to take a look at trying to simplify this outside of this PR, if you agree?

Reply:
Ah, just realised the PR was started 2 years ago :sweat_smile: